server/docs/backend-calls.md

```diff
@@ -5,6 +5,7 @@
 | DELETE | `/user` | deletes user | - | - | ✔️ | `{"Status": "REMOVED", "Error": ""}`| [remove passwords before user](https://github.com/keycloud/keycloud/issues/25) |
 | PUT | `/user` |  updates username | - | - | ✔️ | - | [user updating not working](https://github.com/keycloud/keycloud/issues/26) |
 | GET | `/password` | retrieves specific password | - | `{"username": "johndoe", "url": "john.doe"}` | ✔️ | - | [GET password not working](https://github.com/keycloud/keycloud/issues/28) |
+| GET | `/password-url` | retrieves all passwords and usernames according to provided url | - | `{"url": "john.doe"}` | ✔️ | `[{"Password":"doejohn","Id":"9","Url":"john.doe","Username":"johndoe"},{...}}]` | - |
```

Comment:
maybe consider renaming the call to password-by-url

Reply:
Done.

Reply:
Following the rest idea, you shoud just modiy the "normal" GET /passwords call to be able to search for an url with the optional url parameter

Reply:
@Dudeldu well, the GET `/passwords` is designed to retrieve all passwords in order to render the main KeyCloud password table.
I definitely get point and I want to keep the amount of different routers as low as possible too but still I think differentiating those is the best design.